configure.ac

```diff
@@ -225,6 +282,217 @@ else
+                   [Enable rdtsc measurement of rx ready poll to lwip]),
+    [],
+    [with_rdtsc_rx_ready_poll_to_lwip=no]
+)
+
+AS_IF([test "x$with_rdtsc_rx_ready_poll_to_lwip" == xyes],
+	[AC_DEFINE([RDTSC_MEASURE], 1, [Define to 1 to enable rdtsc measurements.])]
+	[AC_DEFINE([RDTSC_MEASURE_RX_READY_POLL_TO_LWIP], 1, [Define to 1 to enable rdtsc measurement of rx ready poll to lwip.])]
+	[AC_MSG_RESULT([yes])],
+	[AC_MSG_RESULT([no])]
+)
+
+AC_MSG_CHECKING([if rdtsc-rx-lwip-to-receivefrom is enabled])
+AC_ARG_WITH([rdtsc-rx-lwip-to-receivefrom],
+    AC_HELP_STRING([--with-rdtsc-rx-lwip-to-receivefrom],
+                   [Enable rdtsc measurement of rx lwip to receivefrom]),
+    [],
+    [with_rdtsc_rx_lwip_to_receivefrom=no]
+)
+
+AS_IF([test "x$with_rdtsc_rx_lwip_to_receivefrom" == xyes],
+	[AC_DEFINE([RDTSC_MEASURE], 1, [Define to 1 to enable rdtsc measurements.])]
+	[AC_DEFINE([RDTSC_MEASURE_RX_LWIP_TO_RECEVEFROM], 1, [Define to 1 to enable rdtsc measurement of rx lwip to receivefrom.])]
+	[AC_MSG_RESULT([yes])],
+	[AC_MSG_RESULT([no])]
+)
+
+ 
+AC_MSG_CHECKING([if rdtsc-tx-sendto-to-after-post-send is enabled])
+AC_ARG_WITH([rdtsc-tx-sendto-to-after-post-send],
+    AC_HELP_STRING([--with-rdtsc-tx-sendto-to-after-post-send],
+                   [Enable rdtsc measurement of tx sendto to after post send]),
+    [],
+    [with_rdtsc_tx_sendto_to_after_post_send=no]
+)
+
+AS_IF([test "x$with_rdtsc_tx_sendto_to_after_post_send" == xyes],
+	[AC_DEFINE([RDTSC_MEASURE], 1, [Define to 1 to enable rdtsc measurements.])]
+	[AC_DEFINE([RDTSC_MEASURE_TX_SENDTO_TO_AFTER_POST_SEND], 1, [Define to 1 to enable rdtsc measurement of tx sendto to after port send.])]
+	[AC_MSG_RESULT([yes])],
+	[AC_MSG_RESULT([no])]
+)
+
+AC_MSG_CHECKING([if rdtsc-tx-verbs-post-send is enabled])
+AC_ARG_WITH([rdtsc-tx-verbs-post-send],
+    AC_HELP_STRING([--with-rdtsc-tx-verbs-post-send],
+                   [Enable rdtsc measurement of tx verbs post send]),
+    [],
+    [with_rdtsc_tx_verbs_post_send=no]
+)
+
+AS_IF([test "x$with_rdtsc_tx_verbs_post_send" == xyes],
+	[AC_DEFINE([RDTSC_MEASURE], 1, [Define to 1 to enable rdtsc measurements.])]
+	[AC_DEFINE([RDTSC_MEASURE_TX_VERBS_POST_SEND], 1, [Define to 1 to enable rdtsc measurement of tx verbs post send.])]
+	[AC_MSG_RESULT([yes])],
+	[AC_MSG_RESULT([no])]
+)
+
+
+AC_MSG_CHECKING([if rdtsc-receivefrom-to-sendto is enabled])
+AC_ARG_WITH([rdtsc-receivefrom-to-sendto],
+    AC_HELP_STRING([--with-rdtsc-receivefrom-to-sendto],
+                   [Enable rdtsc measurement of receivefrom to sendto]),
+    [],
+    [with_rdtsc_receivefrom_to_sendto=no]
+)
+
+AS_IF([test "x$with_rdtsc_receivefrom_to_sendto" == xyes],
+	[AC_DEFINE([RDTSC_MEASURE], 1, [Define to 1 to enable rdtsc measurements.])]
+	[AC_DEFINE([RDTSC_MEASURE_RECEIVEFROM_TO_SENDTO], 1, [Define to 1 to enable rdtsc measurement of receivefrom to sendto.])]
+	[AC_MSG_RESULT([yes])],
+	[AC_MSG_RESULT([no])]
+)
```

Comment:
It looks as this code related measurements interferes with https://github.com/Mellanox/libvma/pull/194 and probably should be done separate pull request.